docs/source/gbd2017_models/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -0,0 +1,353 @@
+
+
+.. _1.1:
+
+1.1 Project overview
+--------------------
+
+There is belief that benefits of BEP supplementation are currently underestimated and that targeting individual pregnant women with low BMI may be more cost beneficial. There are currently on-going Gates funded clinical trials evaluating the impact of BEP in India, Pakistan, Mali and Tanzania. This simulation project aims to estimate the cost benefits of universal or targeted offer of BEP if the anticipated improvement in infant outcomes are achieved. 
+
+
+.. _1.2:
+
+1.2 Literature review
+---------------------
+
+
+.. _2.0:
+
+2.0 Modeling aims and objectives
+++++++++++++++++++++++++++++++++
+
+To understand the cost-effectiveness of BEP and MMN supplementation for pregnant and lactating women (PLW) on child health outcomes. We will consider the costs and DALYs of baseline, and all scenarios
+
+Objective 1: Comparing the cost-effectiveness of BEP, MMN, and IFA
+This objective will compare MMN scenario vs baseline, and IFA vs MMN scenario
+
+Objective 2:  Comparing the cost-effectiveness of universal vs. targeted BEP administration
+This objective will compare scenario with targeted BEP vs scenario with universal BEP
+
+
+.. _3.0:
+
+3.0 Causal framework
+++++++++++++++++++++
+
+.. _3.1:
+
+3.1 Causal diagram
+------------------
+
+  .. image:: causal_dag_bep_scenarios.svg
+
+**Outcome (O)**:
+
+  - (1) birthweight in grams
+  - (2) LAZ score
+  - (3) WLZ score
+
+
+**Effect modifiers**:
+
+  - The effect of BEP on birthweight is differential according to maternal BMI
+
+:underline:`Current evidence scenarios`
+
+• Under the current evidence scenarios, the intervention BEP affects only birthweight. It does so differentially by maternal BMI status (Ota 2015). BMI is an effect modifier on the relationship between BEP and birthweight. 
+
+• BMI should have an effect on LAZ and WLZ scores suggested by the literature (and Gates) but we are not intervening on BMI, and so simply correlatng birthweight and LAZ / WLZ will capture the relationship between BMI and LAZ/WLZ.
+• note x represents a crude effect size because we are not intervening on BMI and we only use this effect in baseline. (ideally crude mean shift birthweight by maternal BMI)  
```

Comment:
```suggestion
• note x represents a crude effect size because we are not intervening on maternal BMI and we only use this effect to capture the *non-causal* baseline correlation between pre-pregnancy maternal BMI and child birthweight (ideally crude mean shift birthweight by maternal BMI)  
```